libs/sdk-backend-tiger/src/convertors/toBackend/afm/TotalsConverter.ts

```diff
@@ -96,3 +233,30 @@ function convertTotalType(type: TotalType): TotalFunctionEnum {
     // type === "nat"
     throw new Error("Tiger backend does not support native totals.");
 }
+
+function getAttributeBucketIdentifiers(attribute: IAttribute[]) {
+    return attribute.map((a) => bucketItemLocalId(a));
+}
+
+/**
+ * With the new possibility of adding totals/subtotals for columns. Need to be aware when this can Check if this is the case,
+ * to update the execution properly with the missing values to obtain these new values.
+ */
+function shouldAddGrandTotalOrMarginalTotal(
```

Comment:
what about this naming: `hasRowOrColumnTotals`, maybe even something more generic, receiving just an array of buckets in the argument?

Reply:
Have updated the method name, but i'd rather keep the arguments as they are, since they are also used later on.